static/sass/main.scss

```diff
@@ -23,7 +23,7 @@ $breakpoint-navigation-threshold: 960px;
 @include maas-p-tables;
 @include maas-p-contact-modal;
 @include maas-p-strip--dark;
-@include maas-l-docs;
+// @include maas-l-docs;
```

Comment:
If this is no longer necessary, I'd suggest to remove this line and the associated 
`static/sass/_layout_docs.scss` file. 
```suggestion
```